utils/httpUtils.ts

```diff
@@ -27,6 +27,23 @@ export const get = async (url: string, args: any, context: any) => {
     }
 };
 
+export const simpleGet = async (url: string, context: any) => {
```

Comment:
Could we just reuse the `get` function above?  It looks like as long as you don't pass an `args` object with the `snapshotLinkId` key, it should work the same as the `simpleGet` method.